build/dockerfiles/rhel.Dockerfile

```diff
@@ -96,3 +96,13 @@ ENTRYPOINT ["/usr/local/bin/entrypoint.sh"]
 CMD ["/usr/local/bin/rhel.entrypoint.sh"]
 
 # append Brew metadata here
+
```

Comment:
tiny nitpick but the `# append Brew metadata here` should be at the end of the file :)

Reply:
fixed